diff --git a/app/classes/Framadate/Services/MailService.php b/app/classes/Framadate/Services/MailService.php
index f99b1fe..111863d 100644
--- a/app/classes/Framadate/Services/MailService.php
+++ b/app/classes/Framadate/Services/MailService.php
@@ -82,7 +82,7 @@ class MailService {
      * @param PHPMailer $mailer
      */
     private function configureMailer(PHPMailer $mailer) {
-        $mailer->isSMTP();
+        $mailer->isSendmail();
 
         $available_options = [
             'host' => 'Host',
